pkgs/development/libraries/mesa/opencl.patch

```diff
@@ -1,8 +1,20 @@
+From 1036468423866fed446a8ede3ed65b865134050b Mon Sep 17 00:00:00 2001
+From: Cole Mickens <cole.mickens@gmail.com>
+Date: Sun, 5 Nov 2023 17:26:17 -0500
+Subject: [PATCH] opencl.patch
```

Comment:
If there's no description, we should keep this as a diff rather than introduce all the extra noise.

Reply:
Done.